### PR TITLE
Add new section for PhilipsTV binary_sensor

### PR DESCRIPTION
**Pull request by Floyer007**
## Proposed change

A new feature within the PhilipsTV integration requires some little explanation or at least the listing of those new functionality.
This PR will add those new featured binary_sensor.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/94691

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - ~~I made a change to the existing documentation and used the `current` branch.~~
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/philips_js.markdown [hunk 1]
```diff
@@ -17,12 +19,15 @@ ha_platforms:
   - media_player
   - remote
   - switch
+  - sensor
+  - binary_sensor
```
Comment (frenck) [inline source/_integrations/philips_js.markdown]: PRs to the Core codebase may not contain multiple platforms in a single PR, therefore, this PR can also not contain multiple platforms being added in a single PR.

**Review by frenck — DISMISSED** [review]

File: source/_integrations/philips_js.markdown (hunk 1 above)
Comment (Floyer007) [reply source/_integrations/philips_js.markdown]: Sure. Sorry about that. Understood the system now.

I changed both PRs to support binary_sensor only.
A new one will follow for sensor

File: source/_integrations/philips_js.markdown [hunk 2]
```diff
@@ -5,6 +5,7 @@ ha_category:
   - Light
   - Media Player
   - Remote
+  - Binary Sensor
```
Comment (c0ffeeca7) [inline source/_integrations/philips_js.markdown]:
```suggestion
  - Binary sensor
```
We're now doing sentence-style capitalization on the categories, too

File: source/_integrations/philips_js.markdown [hunk 3]
```diff
@@ -141,3 +143,12 @@ overload the event bus in Home Assistant.
 
 Some TV's allow you to sync the processed ambilight color data to your Philips Hue bridge. This will make your Hue lights sync with the TV ambilight without the need to purchase a Hue Play HDMI Sync Box.
 The integration exposes a "Ambilight+Hue" switch entity when your TV supports it which enables you to toggle this.
+
+
+## Binary Sensor
```
Comment (c0ffeeca7) [inline source/_integrations/philips_js.markdown]:
```suggestion
## Binary sensor
```

File: source/_integrations/philips_js.markdown [hunk 4]
```diff
@@ -141,3 +143,12 @@ overload the event bus in Home Assistant.
...
+
+Some newer OS version support live TV recording functions via the API.\
+For those TV'S this integration supports two entities:
```
Comment (c0ffeeca7) [inline source/_integrations/philips_js.markdown]:
```suggestion
Some newer OS versions support live TV recording functions via the API.
For those TVs, this integration supports two entities:
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @Floyer007 :+1:

**Pull request merged**